pkgs/tools/security/krunner-pass/default.nix

```diff
@@ -1,6 +1,6 @@
 { mkDerivation, lib, fetchFromGitHub, fetchpatch, cmake, extra-cmake-modules
 , kauth, krunner
-, pass, pass-otp ? null }:
+, pass }:
```

Comment:
This is one case where this PR actually finds a bug in nixpkgs: This expression was previously called with both `pass` and `pass-otp`, however the latter of which is just an alias of the former with an extension:
https://github.com/NixOS/nixpkgs/blob/b27f145c81e181723304cb7c2972f025b5d5e775/pkgs/top-level/aliases.nix#L556

CI however checks that aliases aren't used by not including them in the scope. The fact that this expression used `? null` allowed it to get around the CI check, because then just the default of `null` was used!

This is notably the only package that is getting rebuilt, and possibly this change breaks the package, depending on which `pass` took priority beforehand.